src/app/src/mainwindow.cpp

```diff
@@ -815,3 +833,21 @@ void MainWindow::acquisitionThread(MainWindow* cls) {
     spdlog::info("Acquisition done, sending signal");
     emit cls->sig_acquisition_done();
 }
+
+
+
+/**
+ * Helper function that will take the line time for each mode from the config file and
+ * calculate the max frame rate based on the caputure reagion height.
+*/
+double MainWindow::calc_max_frame_rate(int p1,int p2,int16_t spdtable_index,double line_times [4],uint16_t spdtable){
+    spdlog::info("Calculating max frame rate");
+
+
+    int number_of_rows = int(abs(p2 - p1));
+    double max_frame_rate = 1000000.0 / (line_times[spdtable] * number_of_rows);
+
+    spdlog::info("Max frame rate is: {},",max_frame_rate);
```

Comment:
The log message doesn't need the trailing `,`, also it's good style to add a space after `,`'s in argument lists, it makes the code easier to read